docs/data/joy/customization/using-css-variables/using-css-variables.md

```diff
@@ -1,81 +1,97 @@
 # Using CSS variables
 
-<p class="description">Learn how to use CSS variables to customize Joy UI's components.</p>
+<p class="description">Learn how to use CSS variables to customize Joy UI components.</p>
 
-## Theme object
+## Introduction
```

Comment:
I think we could add another sentence or two providing more details about _what_ CSS variables are and _why_ you'd want to use them.